plan/conflict_test.go

```diff
@@ -129,7 +129,7 @@ func (suite *ResolverSuite) TestStrictResolver() {
 
 	// legacy record's resource value will not match any candidates resource label
 	// therefore pick minimum again
-	suite.Equal(suite.bar127A, suite.perResource.ResolveUpdate(suite.legacyBar192A, []*endpoint.Endpoint{suite.bar127A, suite.bar192A}), " legacy record's resource value will not match, should pick minimum")
```

Comment:
What is the reason of changing it to NotEqual?

Reply:
Due to the logical change of plan/conflict.go, it should be compatible for single records and multiple records.